client/styles/components/header.css

```diff
@@ -0,0 +1,25 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+@tailwind variants;
+
+.nav {
+  @apply bg-white border-gray-200 px-2 sm:px-4 py-2.5 rounded;
+}
+
+.nav-container {
```

Comment:
same here.

create a container class, to use in different context.

Reply:
hmm make sense, i'm thinking about use these APPLY just for this context and not for many others, ok got it

Reply:
for the cont-1 i named `container-flex`, because on the css line it uses more flex attribuites
and the same thing for the cont-2 i named container-full